File: example/wash.cpp
#include <bits/stdc++.h>

std::string buff;

int main(int argc, char *argv[]) {
    std::ifstream ifs(argv[1]);
    std::ofstream ofs(argv[2]);

    while (ifs >> buff) {
        for (auto c : buff) {
            if (c == ',') ofs << '\t';
            else ofs << c;
        }
        ofs << std::endl;
    }
    return 0;
}

File: example/wash.py
#!/usr/bin/env python3
# wash zoo

import sys

ifs = open(sys.argv[1], "r")
ofs = open(sys.argv[2], "w")
s = '\t'

for line in ifs:
    split_line = line.split()
    split_line = split_line[1:]
    ss = s.join(split_line)
    
    ofs.write("%s\n" % ss)

        
